src/blobresource.js

```diff
@@ -33,8 +33,10 @@ angular.module('hypermedia')
 
     // Prototype properties
     BlobResource.prototype = Object.create(Resource.prototype, {
-      constructor: {value: BlobResource},
+      constructor: {value: BlobResource}
+    });
 
+    BlobResource.prototype = HypermediaUtil.defineProperties(BlobResource.prototype, {
```

Comment:
Do we need to reassign `BlobResource.prototype here`? I believe `HypermediaUtil.defineProperties` modifies it in-place?